enhancements/windows-containers/monitoring-windows-nodes.md

```diff
@@ -0,0 +1,159 @@
+ `openshift-windows-machine-config-operator` namespace so that cluster
+  monitoring stack will pick up the Service Monitor created by WMCO.
+* Adding privileges to WMCO to create Services, Endpoints, Service Monitor in
+  the `openshift-windows-machine-config-operator` namespace.
+* Creating a Service and Endpoints in `openshift-windows-machine-config-operator`
+  namespace that point to windows_exporter endpoint. WMCO will be using default
+  values to define metrics endpoint on windows_exporter which is equivalent to
+  `<internal-ip>:9182/metrics` for a given Windows node. The Endpoints object
+  created in the namespace will consist of subsets of endpoints from all the
+  Windows nodes.
+* Creating a Service Monitor in `openshift-windows-machine-config-operator`
+  namespace for Service created above.
+  
+To display node and workload graphs WMCO is responsible for the following:
+
+* Adding custom Prometheus rules in `openshift-windows-machine-config-operator` 
+  namespace. The custom recording rules will be created using Windows metrics
+  exposed by the windows_exporter and will have the same names as Linux
+  recording rules. This is to make use of same console queries as Linux.
+* Note that WMCO is unable to display workload graphs for the Windows Nodes
+  with current the implementation. See [Risks and Mitigations](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/monitoring-windows-nodes.md#risks-and-mitigations) 
+  for details.
+  
+#### Future Plans
+
+As we move forward our plan to display monitoring graphs is to create a common
+interface for Windows and Linux recording rules that is managed by the
+cluster monitoring operator. WMCO would no longer be responsible for creating
+custom Prometheus rules to display node graphs. The unified interface for
+monitoring will return results for both Linux and Windows nodes for a single
+query. The console queries created as a result of the unified interface will
+be different from the existing queries that would need to be updated. This
+will ensure that we have a consistent user experience for monitoring across
+ Linux and Windows.
+
+### Risks and Mitigations
+
+The main risk with this proposal is renaming Windows metrics to display workload
+graphs. The pod metrics for Linux come from cAdvisor. However, we do not get
+same metrics from cAdvisor for Windows nodes. This becomes a hindrance to
+display pod graphs by creating custom recording rules to use same console 
+queries as Linux workloads.
+
+Mitigation
+ Use metrics exposed by the windows_exporter to display pod graphs as
+ mentioned in the [Future Plans](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/monitoring-windows-nodes.md#future-plans)
+ in detail. This also requires changes in console queries that support OS
+ specific metrics.
+
+## Design Details
+
+### Test Plan
+
+We plan to add e2e tests to ensure
```

Comment:
You should state that this is the current tests and it will cover the future work or mention what is needed for the future work.